src/ResultRenderer/FailureRenderer.php

```diff
@@ -30,6 +30,35 @@ public function __construct(Failure $result)
      */
     public function render(ResultsRenderer $renderer)
```

Comment:
It may not be output that resulted in the failure. I believe this was to catch errors that resulted in `CodeExecutionExceptions`, where the Warnings of the `OutputFailure` was simply if the output did not match the expected.